README.md

```diff
@@ -278,6 +282,42 @@ Example:
     staging: true
 ```
 
+### `verify-enable`
+
+**Default**: `true`
+
+The `verify-enable` setting controls whether or not the generated signatures and certificates are
+verified with the `sigstore verify` subcommand after all files have been signed.
+
+This is not strictly necessary but can act as a smoke test to ensure that all signing artifacts were
+generated properly and the signature was properly submitted to Rekor.
+
+Example:
+
+```yaml
+- uses: trailofbits/gh-action-sigstore-python@v0.0.2
+  with:
+    inputs: file.txt
+    verify-enable: false
+```
+
+### `verify-cert-email`
```

Comment:
Since we know we're signing on GitHub Actions, we might be able to figure out what the SAN should be. So maybe this should just be an on/off switch.

This is an area of active discussion for `cosign` and `sigstore-python` about whether this should be required and what the UX around this should be. So for now, I'm thinking we can just make this a thin wrapper around the existing CLI option and look into this again once a decision about the `cosign` UX has been made.